### DISCUSSION TITLE
Backbone border router

### DISCUSSION THREAD

**Discussion by kmfnjtze8888** [openthread/openthread#8637] [Q&A] [answered]
Hi there,

What is the difference between a _primary_ and _secondary_ backbone border router and how does OpenThread decide which one will be primary or secondary? Does a primary bbr mean the devices will prioritize sending data to it instead of a secondary bbr?

Also, is there a way to modify a border router to become either a primary or secondary?

Thanks.

**Comment by jwhui** [comment, marked as answer]
A Thread Backbone Router is responsible for:
1. Forwarding IPv6 multicast packets between Thread and another IPv6 link.
2. Maintaining state for Thread Devices within a shared Thread Domain (which is a collection of Thread networks that share the same IPv6 prefix). Thread Domain configuration is optional and not currently used by default in OpenThread.

The Primary Backbone Router is the one currently elected to perform the Backbone Router functions above. The Secondary Backbone Routers monitor Thread Network Data and may step in if the Primary Backbone Router is no longer available.

As for normal IPv6 unicast data traffic, the Backbone Router role (primary or secondary) does not affect routing decisions.
